drivers/serial/uart_mspm0g3xxx.c

```diff
@@ -99,21 +99,24 @@ static void uart_mspm0g3xxx_irq_tx_enable(const struct device *dev)
 {
 	const struct uart_mspm0g3xxx_config *config = dev->config;
 
-	DL_UART_enableInterrupt(config->regs, DL_UART_INTERRUPT_TX);
+	DL_UART_enableInterrupt(config->regs, DL_UART_INTERRUPT_TX | DL_UART_INTERRUPT_EOT_DONE);
 }
 
 static void uart_mspm0g3xxx_irq_tx_disable(const struct device *dev)
 {
 	const struct uart_mspm0g3xxx_config *config = dev->config;
 
-	DL_UART_disableInterrupt(config->regs, DL_UART_INTERRUPT_TX);
+	DL_UART_disableInterrupt(config->regs, DL_UART_INTERRUPT_TX | DL_UART_INTERRUPT_EOT_DONE);
 }
 
 static int uart_mspm0g3xxx_irq_tx_ready(const struct device *dev)
 {
 	const struct uart_mspm0g3xxx_config *config = dev->config;
 
-	return (DL_UART_getEnabledInterruptStatus(config->regs, DL_UART_INTERRUPT_TX)) ? 0 : 1;
+	return (DL_UART_getEnabledInterruptStatus(config->regs, DL_UART_INTERRUPT_TX |
+									DL_UART_INTERRUPT_EOT_DONE))
+		       ? 1
+		       : 0;
```

Comment:
Something happened here. I think spaces instead of tabs in one of the lines or something?

Might make sense to declare a variable and set the value through some `if` rather than ternary

Reply:
Clang format - I don't manually format anything :)

Reply:
But still, the formatting looks off.